target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -74,7 +74,6 @@ endef
 
 define Device/ubnt-sw
   $(Device/ubnt)
-  DEVICE_VARIANT := SW
```

Comment:
Please remove that in the appropriate early commit where you add the definition.

Reply:
Ah, thanks. Seems to have slipped in there during rebase